### ISSUE TITLE
Version with speeds only up to 4x

### ISSUE THREAD

**Issue by BloodRaven0** [sk7725/TimeControl#37]
TLDR: Please release a version of this mod that only has up to 4x or maximum 8x.

Hey, so I was defending a sector with this mod installed, and I was microing some units and I accidentally misclicked and set the time to 256x. I corrected the mistake half a second later, but the damage was already done.  This fast-forwarded time to about 10 waves later, and there were enemies past my defenses attacking my core. I basically lost hours of progress due to a stupid misclick. I have disabled this mod for future play.

Basically, above 4x speed, this mod breaks the game on MANY levels, so I don't see the point of having these options, especially when things like this could happen.
Can we get a version of this mod that only has 0.25x, 0.5x, 1x, 2x and 4x? I think those speeds cover about 99% of this mods' usage and won't break the game TOO badly.
(Also I liked the old UI that you only had to click on, not the slider one but that's fine lol)

**Comment by BloodRaven0** [comment]
I forked this mod to make this "lite" version that I proposed: https://github.com/BloodRaven0/TimeControlLite

**Comment by jw-taylor** [comment]
> I forked this mod to make this "lite" version that I proposed: https://github.com/BloodRaven0/TimeControlLite

Can't see how to raise an issue on your fork but just FYI on mobile you can't minimise the slider, same issue as this original mod

**Comment by BloodRaven0** [comment]
@jw-taylor I had no idea that minimizing it was even a thing. Was there such a functionality in older versions?

**Comment by jw-taylor** [comment]
> I forked this mod to make this "lite" version that I proposed: https://github.com/BloodRaven0/TimeControlLite

Can't see how to raise an issue on your fork but just FYI on mobile you can't minimise the slider, same issue as this original mod 

> @jw-taylor I had no idea that minimizing it was even a thing. Was there such a functionality in older versions?

The in-game description of the original mod says "press & hold 'x1' to fold the buttons" which I interpret as long-press x1 to minimise the slider (which would make sense, because on mobile the slider covers the build menu). I think there's an open issue for the original mod which mentions that so I'm not the only person that thinks that's what that means.
